roles/branch-validation/tasks/main.yml

```diff
@@ -71,6 +71,28 @@
     wait: yes
   with_items: groups[instance_group]
   register: instances
+  when: not groups['compute'] | intersect(groups['controller']) | length | bool
```

Comment:
So, it runs from `controller[0]` but it attempts to schedule instances on each AZ, and this, I think, really all depends on whether or not it can schedule to a control node.  So, AFAICT, when I read the original playbook, it was trying to a) launch an instance on each compute node and b) if there were converged controllers, schedule on each converged control node explicitly.

I made some guesses based on my interpretation of the intent of the original playbook so this could be...wrong.  IDK.